code/modules/mob/living/carbon/monkey/combat.dm

```diff
@@ -158,7 +158,7 @@
 	switch(mode)
 		if(MONKEY_IDLE)		// idle
 			if(enemies.len)
-				var/list/around = view(src, MONKEY_ENEMY_VISION) // scan for enemies
+				var/list/around = view(MONKEY_ENEMY_VISION, src) // scan for enemies
```

Comment:
You accidentally put the arguments for view in the wrong order.

Reply:
So, I think it might actually work both ways. I saw some spots in the code where it was view(object, range) and it seemed to work.

Reply:
It does work both way, ref recommends your way though.